<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>351</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>351</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Поиск медикаментов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>27</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QSplitter" name="splitter">
         <property name="minimumSize">
          <size>
           <width>481</width>
           <height>326</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>481</width>
           <height>326</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>481</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>481</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Введите заболевание или симптом (через запятую)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="findMed">
          <property name="minimumSize">
           <size>
            <width>351</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>351</width>
            <height>20</height>
           </size>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget">
          <property name="minimumSize">
           <size>
            <width>431</width>
            <height>191</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>431</width>
            <height>191</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>441</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>441</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Введите медикамент, который хотите назначить пациенту&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="minimumSize">
           <size>
            <width>431</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>431</width>
            <height>25</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="appMedName">
             <property name="minimumSize">
              <size>
               <width>297</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>297</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="appBtn">
             <property name="minimumSize">
              <size>
               <width>126</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>126</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Назначить медикамент</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="recHistoryBtn">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>111</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>История рецептов</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>findButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
